app/src/main/java/com/example/pool/dto/Algae.kt

```diff
@@ -31,16 +31,16 @@ class Algae(val type: String, val hoursCantSwim: Float, val ozPerGallon: Float,
      * @override toString function
      * @return chemical name and a small summary about the chemical
      */
+
     @Override
     override fun toString(): String {
-        //may need to be modified for units
         if (this.hoursCantSwim > 0 ) {
-            return this.type + " should never appear, and any amount is dangerous. " + this.ozPerGallon + " of algecide and " +
-            this.chlBoostPerGallon + " of chlorine is necessary, and isn't safe to be in the pool for " + hoursCantSwim + " after use."
+            "There are more than 21,000 types of algae, typically classified by color. Alkalinity for pool water should be between 7.4 and 7.6, and anywhere between 80 and 140 ppm. " + return this.type + " should never appear, and any amount is dangerous. " + this.ozPerGallon + " of algecide and " +
+            this.chlBoostPerGallon + " of chlorine are necessary are necessary to both shock and treat the system, and it isn't safe for pool goers for at least " + this.hoursCantSwim + " after use."
```

Comment:
How can you internationalize this?  For this much text, I'd pull it from a service.  That allows you to iterate and pass a locale from the client, to get a localized String in return.  Hardcoding it is risky, as it's hard to change, and hard to internationalize.